packages/pharos/src/components/header/pharos-header.test.ts

```diff
@@ -33,19 +34,15 @@ describe('pharos-header', () => {
               <test-pharos-dropdown-menu-item link="/action/showJournals?browseType=title"
                 >by Title</test-pharos-dropdown-menu-item
               >
-              <test-pharos-dropdown-menu-item link="/site/collection-list"
-                >by Collections</test-pharos-dropdown-menu-item
-              >
-              <test-pharos-dropdown-menu-item link="/publishers"
-                >by Publisher</test-pharos-dropdown-menu-item
-              >
+              <test-pharos-dropdown-menu-item link="/publishers">by Publisher</pharos-dropdown-menu-item>
             </test-pharos-dropdown-menu>
-            <test-pharos-dropdown-menu-nav-link
-              href="/account/workspace"
+            <test-pharos-dropdown-menu-nav-category
               id="tools-link"
               data-dropdown-menu-id="tools-menu"
               data-dropdown-menu-hover
-              >Tools</test-pharos-dropdown-menu-nav-link
+              >
+                <span slot="category">Browse</span>
```

Comment:
Should this be `Tools`?

Reply:
This is just a test with arbitrarily populated values, not meant to reflect the actual header on the site.